packages/lib-classifier/src/components/Classifier/components/MetaTools/components/CollectionsButton/CollectionsButton.spec.js

```diff
@@ -0,0 +1,24 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import sinon from 'sinon'
+import { PlainButton } from '@zooniverse/react-components'
+
+import CollectionsButton, { Collect } from './CollectionsButton'
+
+let wrapper
+
+describe('Component > CollectionsButton', function () {
+  before(function () {
+    wrapper = shallow(<CollectionsButton />)
+  })
+
+  it('should render without crashing', function () {
+    expect(wrapper).to.be.ok
+  })
+
+  it('should display a Collect icon', function () {
+    const button = wrapper.find(PlainButton)
+    const { icon } = button.props()
+    expect(icon).to.deep.equal(<Collect />)
+  })
```

Comment:
Should there be a test for correctly calling the `onClick` prop on click?

Reply:
Good point.